README.md

```diff
@@ -11,3 +11,47 @@ Health Check
 ---
 
 To see your applications health enter url `http://localhost:8081/healthcheck`
```

Comment:
You've mentioned the healthcheck URL, but not the actual localhost URL (could also add the AWS link as well)